invenio_communities/views/communities.py

```diff
@@ -216,6 +216,22 @@ def communities_settings_privileges(pid_value, community, community_ui):
     )
 
 
+@pass_community(serialize=True)
+def communities_settings_curation_policy(pid_value, community, community_ui):
+    """Community settings/curation-policy page."""
+    permissions = community.has_permissions_to(
+        ["update", "read", "search_requests", "search_invites"]
```

Comment:
Minor: Do we need to check for all these permissions? wouldn't be enough the "update"? 

Same question for the functions below.

Reply:
Will test! 🙏